tests/python/multidevice/README.md

```diff
@@ -0,0 +1,12 @@
+<!--
+ * SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+-->
+
+# How to run multidevice tests?
+
+1. Install NvFuser with distributed supported enabled. Set `export NVFUSER_BUILD_WITHOUT_DISTRIBUTED=0`.
+2. Check that `NVFUSER_DISABLE` environment variable does not contain `multidevice`. Do not `export NVFUSER_DISABLE=multidevice`.
+3. Install test specific dependencies with `pip install pytest-mpi mpi4py`
```

Comment:
How about a similar structure like https://github.com/Lightning-AI/lightning-thunder/tree/main/requirements and adding pytest-mpi and mpi4py over there?